types/react-loadable/index.d.ts

```diff
@@ -24,64 +25,76 @@ export interface CommonOptions {
      * (this is considered a valid React component).
      */
     // NOTE: () => null is only needed until React.SFC supports components returning null
-    LoadingComponent: React.ComponentClass<LoadingComponentProps> | React.SFC<LoadingComponentProps> | (() => null);
+    loading: React.ComponentClass<LoadingComponentProps> | React.SFC<LoadingComponentProps> | (() => null);
```

Comment:
Same here, can be `React.ComponentType<LoadingComponentProps>`. Not sure if the `(() => null)` is still needed, but if `loading: () => null` doesn't pass without it then it is.